README.md

```diff
@@ -98,25 +168,33 @@ and
 <%= javascript_pack_tag 'shop/orders' %>
 ```
 
-## Advanced Configuration
+### Paths
 
 By default, webpacker offers simple conventions for where the webpack configs, javascript app files and compiled webpack bundles will go in your rails app,
 but all these options are configurable from `config/webpack/paths.yml` file.
 
+Suppose, you want
+to change the source directory from `app/javascript` to `frontend` and entry directory from `packs` to `modules` this is how you would do it.
+
 ```yml
 # config/webpack/paths.yml
-source: app/javascript
-entry: packs
+source: frontend
+entry: modules
 output: public
-config: config/webpack
-node_modules: node_modules
 ```
 
-*Note:* Behind the scenes, webpacker will use same `entry` directory name inside `output`
-directory to emit bundles. For ex, `public/packs`
+Please note that you always want to change the source directory.
```

Comment:
I'm not sure this is a typo or you mean something I do not quite get. If I always want to change the source directory then what purpose does a default serve?